Sources/Butter/Router.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-public typealias NetworkCompletion = (Result<Data?, Error>) -> ()
+public typealias NetworkCompletion<T: Decodable> = (Result<T, Error>) -> ()
```

Comment:
what about endpoints that vend an empty success - void is can't conform to decodable afaik

Reply:
i was wondering about that. i'll investigate that

Reply:
You can do an empty Decodable object here, i'll commit another endpoint that has the example.

Reply:
94f397d